src/components/alert/alert/alert.stories.ts

```diff
@@ -166,6 +166,7 @@ const defaultArgs: Args = {
   target: undefined,
   rel: undefined,
   'accessibility-label': undefined,
+  animation: animation.options[0],
```

Comment:
hmm with our new default set to none, it's difficult to see the animation in the storybook.

Should, maybe as we have the prop now, the standard still be "open" respect. "all" for notification? Not sure :/
@kyubisation

Reply:
would be a good compromise to solve it storybook. for the alert we so far don't have an "add alert" button, but we could add it.

Reply:
However, for the alert, I opted to just have `animation='open'` as the default since it is a simpler case. Maybe we could reconsider it when we will implement the close animation

Reply:
I vote for 'all'/'open' for component default, but I'm willing to compromise 😄

Reply:
I misunderstood @kyubisation request :(

I've changed he components default to all/open